sdk/ai/ai-projects/src/agents/index.ts

```diff
@@ -155,6 +155,9 @@ export interface AgentsOperations {
   ) => Promise<FileListResponseOutput>;
   /** Uploads a file for use by other operations. */
   uploadFile: (data: ReadableStream | NodeJS.ReadableStream, purpose: FilePurpose, fileName?: string, requestParams?: OptionalRequestParameters
+  ) => Promise<OpenAIFileOutput>
+    /** Uploads a file for use by other operations. */
+  uploadFileAndPoll: (data: ReadableStream | NodeJS.ReadableStream, purpose: FilePurpose, sleepInterval: number, fileName?: string, requestParams?: OptionalRequestParameters
```

Comment:
Make `sleepInterval` optional since there is a default

Reply:
Made this optional